RandomStudio/random-studio
# Hide volume toggle when no audio track

When a video does not have an audio track, we should hide the Sound on/off volume toggle

**Steps to test**
1.  Open a project which has videos without audio, eg: Dom Perignon
2. The videos on the project page should **not** say Sound On
3. Open a video focus, this should also **not** say Sound On/Off
4. Check that the volume toggle exists on a video with audio
